hibernate-core/src/main/java/org/hibernate/cfg/Ejb3Column.java

```diff
@@ -566,7 +566,7 @@ public void forceNotNull() {
 					}
 
 					final String columnName;
-					if ( "".equals( col.name() ) ) {
+					if ( col.name().isEmpty() ) {
```

Comment:
This used to be null safe and is not anymore. I would keep the null safe behavior just to be sure.